twindle-cli/src/twitter/twitter.js

```diff
@@ -52,25 +51,18 @@ async function getTweetsFromUser(screenName, token) {
   return [finalTweetsData];
 }
 
-/**
- * @param {string} ids
- * @param {string} token
- */
 const getTweetsFromThreads = async (ids, token) => {
-  let { data: responseJSON, status } = await getTweetById(ids, token);
```

Comment:
I need to ask you to revert these back, if you can.

Or you could just copy paste the entire file https://github.com/twindle-co/twindle/blob/main/twindle-cli/src/twitter/twitter.js

Reply:
Or just remove this file from this PR.  I don't see anything new happening in it.